updated hero blurb

aj-bhatia opened the pull request:
# Scope

Updated hero blurb on sponsorship site according to the new Figma outline. [Github](https://github.com/orgs/CruzHacks/projects/2?pane=issue&itemId=40242116)

# Implementation

I copied and pasted the updated blurb from Figma...

# How to Test

- Run the web page
- Look at the new description
- Ensure its correct

aj-bhatia committed:
updated hero blurb

xxzbuckxx commented:
Looks great so far Ajay! It would be great if you could address these 2 things as well:

- update the blurb above the mailing list in accordance to the figma
- Change the devpost button to link to the sponsorship packet

<img width="1441" alt="Screen Shot 2023-10-09 at 5 56 38 PM" src="https://github.com/CruzHacks/cruzhacks-2024-sponsorship/assets/20495250/75f30419-fed5-4482-8ed9-39c215e46cb8">

You will need to slack sponsorship to ask if they are done with the sponsorship packet, or if they have a way they want it to be linked (preferably they would want to link it as a public document in google drive similar to [our participation agreement](https://docs.google.com/document/d/1298VyEqrSIvbzM3U8nvLQsLtY78Vi2mbndr3d5w0UKk/edit?usp=sharing) but they may have a better idea in mind).

zacktraczyk reviewed (changes requested):
Looks great so far Ajay! It would be great if you could address these 2 things as well:

- update the blurb above the mailing list in accordance to the figma
- Change the devpost button to link to the sponsorship packet

<img width="1441" alt="Screen Shot 2023-10-09 at 5 56 38 PM" src="https://github.com/CruzHacks/cruzhacks-2024-sponsorship/assets/20495250/75f30419-fed5-4482-8ed9-39c215e46cb8">

You will need to slack sponsorship to ask if they are done with the sponsorship packet, or if they have a way they want it to be linked (preferably they would want to link it as a public document in google drive similar to [our participation agreement](https://docs.google.com/document/d/1298VyEqrSIvbzM3U8nvLQsLtY78Vi2mbndr3d5w0UKk/edit?usp=sharing) but they may have a better idea in mind).

aj-bhatia committed:
updated sponsorship link and button, as well as the date and location:
q
;

aj-bhatia commented:
Added changes to mailing list blurb and the sponsor us button and link. Additionally, updated the date and location, as well as the styling for the font of the text.

zacktraczyk committed:
fix formatting
